packages/dd-trace/src/plugins/ci_plugin.js

```diff
@@ -16,6 +16,9 @@ module.exports = class CiPlugin extends Plugin {
     super(...args)
 
     this.addSub(`ci:${this.constructor.name}:itr-configuration`, ({ onDone }) => {
+      if (!this.tracer._exporter.getItrConfiguration) {
```

Comment:
It looks like `NoopTracer` does not initialize the `this._exporter`, shouldn't be necessary to check `tracer._exporter` as well?
```suggestion
      if (!this.tracer._exporter || !this.tracer._exporter.getItrConfiguration) {
```

Reply:
yeah good catch